config/settings.yml

```diff
@@ -80,8 +80,6 @@
   :scan_via_host: true
   :use_vim_broker: true
   :use_vim_broker_ems: true
-:container_scanning:
```

Comment:
@simon3z do we want shared stuff here or in manageiq-providers-kubernetes ?
(also the job timeout and the `container_entities` section)